legotrain_helpers.sh

```diff
@@ -0,0 +1,28 @@
+package: legotrain_helpers
+version: "%(tag_basename)s"
+tag: "v1.0.0"
+source: https://github.com/mfasDa/legotrain_helpers
```

Comment:
If this is useful, we should move it under alisw / AliceO2Group, I guess (and give you maintain rights)

Reply:
The package is intended for the current train, therefore alisw would be the most appropriate. The run3 train test structure might be different. The scripts could also go to AliPhysics directly in case it would get a bin directory. This would eventually be even easier.

The goal of the package is to download the train configuration and run the test locally. This is intended for debugging and could be useful for many people in the collaboration in order to develop and test their code locally on existing train configurations. I am using this myself for debugging and would like to share this with whoever is interested.

Reply:
If the only scope of the package is AliPhysics, then I would propose to add it there, actually...